src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -110,13 +110,16 @@ public partial class TxPool : ITxPool, IDisposable
             _filterPipeline.Add(new TooFarNonceFilter(txPoolConfig, _accounts, _transactions, _logger));
             _filterPipeline.Add(new TooExpensiveTxFilter(_headInfo, _accounts, _logger));
             _filterPipeline.Add(new FeeToLowFilter(_headInfo, _accounts, _transactions, _logger));
+            _filterPipeline.Add(new NotEnoughBalanceFilter(_headInfo, _accounts, _logger));
```

Comment:
How does this filter differ from TooExpensiveTxFilter?

Reply:
We will discuss it with @MansurovB-source and @marcindsobczak